src/OSPSuite.Presentation/Presenters/Charts/ChartEditorPresenter.cs

```diff
@@ -192,11 +196,13 @@ public class ChartEditorPresenter : AbstractCommandCollectorPresenter<IChartEdit
       private readonly IChartSettingsPresenter _chartSettingsPresenter;
       private readonly IChartExportSettingsPresenter _chartExportSettingsPresenter;
       private readonly ICurveSettingsPresenter _curveSettingsPresenter;
+      private readonly ICurveColorGroupingPresenter _curveColorGroupingPresenter;
       private readonly IDataBrowserPresenter _dataBrowserPresenter;
       private readonly IChartTemplateMenuPresenter _chartTemplateMenuPresenter;
       private readonly IChartUpdater _chartUpdater;
       private readonly IEventPublisher _eventPublisher;
       private readonly IDimensionFactory _dimensionFactory;
+      private List<DataRepository> _activeObservedData;
```

Comment:
I don't understand what the active observed Data is?

Reply:
this is a list that contains all the observed data that correspond to curves active in the chart. This is only in case the chart has metadata, and it initially gets set at the initialization of the PK-Sim ChartPresenter ( through SetAvailableColorGroupingMetaData(), that gets called from PK-Sim). This is done because the ChartPresenter in Core has no knowledge of the datarepositories corresponding to the curves.

Reply:
hum...but curve as xData and yData with a REpositoru method in it to access the repo...